infra/chromadb_compaction/tests/test_smoke.py

```
@@ -0,0 +1,168 @@
+        response = LambdaResponse(
+            status_code=200, message="Test message", processed_messages=5
+        )
+
+        result_dict = response.to_dict()
+
+        assert result_dict["statusCode"] == 200
+        assert result_dict["message"] == "Test message"
+        assert result_dict["processed_messages"] == 5
+
+        # Test that frozen dataclass prevents modification
+        with pytest.raises(Exception):  # Should be frozen
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)